src/System.Security.Cryptography.X509Certificates/tests/PfxTests.cs

```diff
@@ -104,6 +105,59 @@ public static void ExportWithPrivateKey()
             }
         }
 
+        [Fact]
+        public static void ReadECDsaPrivateKey_WindowsPfx()
+        {
+            using (var cert = new X509Certificate2(TestData.ECDsaP256_DigitalSignature_Pfx_Windows, "Test"))
```

Comment:
Are we testing that this "Windows" pfx can be opened on any platform?  Or rather than having the isOSPlatform check below, should this be [PlatformSpecific]?